python/ray/__init__.py

```diff
@@ -161,9 +161,10 @@ def _ray_user_setup_function():
             m = __import__(module_name, globals(), locals(), [fn_name])
             getattr(m, fn_name)()
         except Exception as e:
-            logger.exception(
-                f"Failed to run user setup function: {user_setup_fn}. "
-                f"Error message {e}")
+            # We still need to allow ray to be imported, even there is
+            # something in the setup function.
+            logger.info(f"Failed to run user setup function: {user_setup_fn}. "
```

Comment:
@pcmoritz , doesn't info show to the user? should we do debug instead?

Reply:
I think in the meeting, we decided to show it to the user. Instead of the exception log with a backtrace, we only showed one line.

Reply:
@iycheng Actually let's do logger.warning here, it is good to indicate that something **could** be wrong here. Also this will soon go away once everybody has the right dependencies installed so the function is actually found :)